packages/kernel-5.15/kernel-5.15.spec

```diff
@@ -107,8 +107,8 @@ Summary: Header files for the Linux kernel for use by glibc
 rpmkeys --import %{S:1} --dbpath "${PWD}/rpmdb"
 rpmkeys --checksig %{S:0} --dbpath "${PWD}/rpmdb"
 rm -rf "${PWD}/rpmdb"
-rpm2cpio %{S:0} | cpio -iu {,./}linux-%{version}.tar {,./}config-%{_cross_arch} {,./}"*.patch" {,./}kernel.spec
-tar -xof linux-%{version}.tar; rm linux-%{version}.tar
+rpm2cpio %{S:0} | cpio -iu {,./}linux-%{version}.tar.xz {,./}config-%{_cross_arch} {,./}"*.patch" {,./}kernel.spec
+tar -xof linux-%{version}.tar.xz; rm linux-%{version}.tar.xz
```

Comment:
I wonder if we can make this line smarter. We are flipping back and forth between the two

- From no compresseion to `xz`: https://github.com/bottlerocket-os/bottlerocket-kernel-kit/commit/9e1f3309e04943600d7ace9b900e0c5ff2c4947e
- From `xz` to no compression:  https://github.com/bottlerocket-os/bottlerocket-kernel-kit/commit/70c3a86dfa3898aba24cefbca8e5a5b0d8db7b43
- And now this again